src/components/Carousel/Carousel.tsx

```diff
@@ -108,6 +110,10 @@ export const Carousel: FC<CarouselProps> = ({
 
   const handleDragging = (dragging: boolean) => () => setIsDragging(dragging);
 
+  useEffect(() => {
+    !!onSlide && onSlide(activeItem);
```

Comment:
Why do you add two `!!`? Isn't just `onSlide` enough?

Reply:
@rluders !! is not necessary and i just added it for readability sake, i can remove it if you prefer.

Reply:
@paghar yes, please make it simplier, just `onSlide && onSlide(activeItem)` is good.